src/lib/__tests__/helpers.test.ts

```diff
@@ -65,6 +65,16 @@ describe("toKey", () => {
     ).toBe("foo/bar?size=10&sleep=false&sort=asc")
   })
 
+  it("does not include undefined options in path", () => {
```

Comment:
We call this method in our loaders when generating paths, unspecified options are passed through to loader as undefined but ignored when adding params to path:
https://github.com/artsy/metaphysics/blob/master/src/lib/loaders/api/loader_interface.ts#L21